public/lessons/db/mongodb.md

```diff
@@ -67,26 +58,80 @@ this is analogous to a *table* in SQL
 
 # Concept: Document
 
-In MongoDB, a *document* is essentially a single JavaScript object
+In MongoDB, a *document* is essentially a single JavaScript *object*
 
-Like in a relational database, a document can be created, read, updated, deleted, indexed, searched for, ...
+Like in a relational database, a document can be *created, read, updated, deleted, indexed, searched for*, ...
 
 *Unlike* in a relational database, a document can contain *any* value for its fields, *including arrays and nested objects*.
 
 This nesting and type-flexibility makes it very appropriate to store whatever JavaScript objects your app uses, without needing to devise a *mapping* between nested objects and joined relational tables.
 
-# Lab: Mongo CLI
+# Do it with JavaScript!
```

Comment:
Let's insert with compass before we jump into JS connections so they get a feel for the structure of the database. Also, append the word "Lab:" to this so that our parser will auto-build the lab link for this section.
Edit: WE can just move the "Explore with Compass" Lab up here, and it should be good.